newsrevenuehub/rev-engine
# NRH-241:  add ga v3 trackers

#### What's this PR do?

- Adds `org_google_analytics_id` and `org_google_analytics_domain` to `RevenueProgram` model
- Tracks all page views to Hub GA v3 account 
- Tracks page views to public-facing donation and thank you pages for respective org, if that org has values set for `org_google_analytics_id` and `org_google_analytics_domain`
- 
#### How should this be manually tested?

- Add `org_google_analytics_id` and `org_google_analytics_domain` to an org's revenue program model, then visit a donation page from that org.  There will be calls to google analytics `collect` end point with pageview data and the Org's GA id.
- Visit any page, and in network tab, will see GA `collect` endpoint being hit up with pageview data and the Hub's GA id.
- Additionally, can check in Google Analytics dashboard

#### Do any changes need to be made before deployment to staging? production? (adding environment variables, for example)?

No.
